cypress/support/fragments/data_import/mapping_profiles/mappingProfileDetails.js

```diff
@@ -67,5 +78,10 @@ export default {
           .click());
       }
     ));
+  },
+
+  checkErrorMessageIsPresented:(textFieldName) => {
+    cy.do(TextField(textFieldName).click());
+    cy.expect(TextField(including(textFieldName)).has({ error: 'Please enter a value' }));
```

Comment:
you can work with this object TextField(textFieldName) without using "including" and move it to const variable in this method

Reply:
Done